set.mm

```diff
@@ -425601,82 +425741,145 @@ that the number n of vertices in G is exactly k(k-1)+1.".  Variant of
     numclwwlkovfel2.e $e |- E = ( Edg ` G ) $.
     $( Properties of an element of the value of operation ` F ` .  (Contributed
        by Alexander van der Vekens, 20-Sep-2018.)  (Revised by AV,
-       28-May-2021.) $)
-    numclwwlkovfel2 $p |- ( ( G e. USGraph /\ N e. NN /\ X e. V )
-                            -> ( W e. ( X F N )
+       28-May-2021.)  (Revised by AV, 13-Feb-2022.) $)
+    numclwwlkovfel2 $p |- ( ( N e. NN /\ X e. V ) -> ( W e. ( X F N )
                      <-> ( ( W e. Word V /\ A. i e. ( 0 ..^ ( ( # ` W ) - 1 ) )
                                      { ( W ` i ) , ( W ` ( i + 1 ) ) } e. E
                              /\ { ( lastS ` W ) , ( W ` 0 ) } e. E )
                            /\ ( # ` W ) = N /\ ( W ` 0 ) = X ) ) ) $=
-      ( wcel w3a co cc0 cfv wceq cusgr cn cv cclwwlksn cword c1 caddc cpr chash
-      crab cmin cfzo wral clsw numclwwlkovf ancoms 3adant1 isclwwlksnx 3ad2ant2
-      eleq2d wa wb anbi1d fveq1 eqeq1d elrab df-3an 3bitr4g bitrd ) GUAOZHUBOZK
-      IOZPZJKHFQZOJRAUCZSZKTZAHGUDQZUJZOZJIUEOCUCZJSWAUFUGQJSUHEOCRJUISZUFUKQUL
-      QUMJUNSRJSZUHEOPZWBHTZWCKTZPZVMVNVSJVKVLVNVSTZVJVLVKWHABDFGHIKLUOUPUQUTVM
-      JVROZWFVAWDWEVAZWFVAVTWGVMWIWJWFVKVJWIWJVBVLCEGHIJMNURUSVCVQWFAJVRVOJTVPW
-      CKRVOJVDVEVFWDWEWFVGVHVI $.
+      ( wcel wa co cc0 cfv wceq cn cv cclwwlksn crab cword caddc cpr chash cmin
+      c1 cfzo wral clsw numclwwlkovf ancoms eleq2d wb isclwwlksnx adantr anbi1d
+      w3a fveq1 eqeq1d elrab df-3an 3bitr4g bitrd ) HUAOZKIOZPZJKHFQZOJRAUBZSZK
+      TZAHGUCQZUDZOZJIUEOCUBZJSVRUJUFQJSUGEOCRJUHSZUJUIQUKQULJUMSRJSZUGEOVAZVSH
+      TZVTKTZVAZVJVKVPJVIVHVKVPTABDFGHIKLUNUOUPVJJVOOZWCPWAWBPZWCPVQWDVJWEWFWCV
+      HWEWFUQVICEGHIJMNURUSUTVNWCAJVOVLJTVMVTKRVLJVBVCVDWAWBWCVEVFVG $.
 
     $d V w $.
-    $( Value of operation ` F ` for argument 2.  (Contributed by Alexander van
-       der Vekens, 19-Sep-2018.)  (Revised by AV, 28-May-2021.) $)
-    numclwwlkovf2 $p |- ( ( G e. USGraph /\ X e. V )
-                          -> ( X F 2 ) = { w e. Word V | ( ( # ` w ) = 2
-                /\ { ( w ` 0 ) , ( w ` 1 ) } e. E /\ ( w ` 0 ) = X ) } ) $=
-      ( wcel wa c2 co cfv wceq crab anbi1i bitri cusgr cc0 cv cclwwlksn cpr w3a
-      chash c1 cword cn simpr numclwwlkovf sylancl cvtx cedg wb clwwlksn2 anass
-      2nn a1i df-3an ancom eqcomi wrdeqi eleq2i anbi12i anbi2i syl6bb rabbidva2
-      3bitr4i eqtrd ) FUALZHGLZMZHNEOZUBAUCZPZHQZANFUDOZRZVPUGPNQZVQUHVPPUEZDLZ
-      VRUFZAGUIZRVNVMNUJLVOVTQVLVMUKUSABCEFNGHIULUMVNVRWDAVSWEVNVPVSLZVRMZWAVPF
-      UNPZUIZLZWBFUOPZLZUFZVRMZVPWELZWDMZWGWNUPVNWFWMVRFVPUQSUTWOWAWCMZMZVRMWOW
-      QVRMZMWNWPWOWQVRURWMWRVRWMWOWAMZWCMZWRWMWAWJMZWLMXAWAWJWLVAXBWTWLWCXBWJWA
-      MWTWAWJVBWJWOWAWIWEVPWHGGWHJVCVDVESTWKDWBDWKKVCVEVFTWOWAWCURTSWDWSWOWAWCV
-      RVAVGVJVHVIVK $.
+    $( Value of operation ` F ` for the second operand being 1.  (Contributed
+       by AV, 11-Feb-2021.) $)
```

Comment:
Maybe "set of closed walks of length one starting from vertex `v`" would be clearer?